### DISCUSSION TITLE
How to display a single Bitmap in a specified coordinate range

### DISCUSSION THREAD

**Discussion by xiaoyan159** [mapsforge/vtm#900] [Q&A] [answered]
If I know the bitmap leftTop and rightBottom coordinates，is there a way to display it on vtm map？ 

I think there are two ways，one of these is modify the BitmapRender，and another is use BitmapTileLayer， but the tileSource is not extend UrlTileSource，we must override the tileSource and tileDataSource，modify the query function。

I have implemented the second one， I wonder if VTM needs this？

**Comment by devemux86** [comment, marked as answer]
You can display any bitmap with a marker overlay (billboard or not).
(see the [MarkerOverlayActivity](https://github.com/mapsforge/vtm/blob/master/vtm-android-example/src/org/oscim/android/test/MarkerOverlayActivity.java) example)
